lib/irb/context.rb

```diff
@@ -48,8 +48,13 @@ def initialize(irb, workspace = nil, input_method = nil)
       end
       if IRB.conf.has_key?(:USE_MULTILINE)
         @use_multiline = IRB.conf[:USE_MULTILINE]
-      elsif IRB.conf.has_key?(:USE_REIDLINE) # backward compatibility
-        @use_multiline = IRB.conf[:USE_REIDLINE]
+      elsif IRB.conf.has_key?(:USE_RELINE) # backward compatibility
+        @use_multiline = IRB.conf[:USE_RELINE]
+      elsif IRB.conf.has_key?(:USE_REIDLINE)
+        warn <<~MSG.strip
+          USE_REIDLINE is deprecated, please use USE_RELINE instead.
+        MSG
+        @use_multiline = IRB.conf[:USE_RELINE]
```

Comment:
If this is for backward compatibility, shouldn't this be `USE_REIDLINE`?

Reply:
maked sense.